README.md

```diff
@@ -55,7 +55,7 @@ A plan is defined like this:
         path pool/dataset3
 
         keep latest 10
-        protect </etc/zfs-cleaner.protected
+        protect /etc/zfs-cleaner.protected
```

Comment:
This is not a typo.

`<` is used to denote a *file* containing a list of snapshot names to protect. It could use some documentation thou.

Reply:
Ah, ok. Got it. Then let's put your comment in the documentation ;)